app/js/enhancedDate.js

```diff
@@ -1,5 +1,52 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function() {    
+    var myDate = new Date();    
```

Comment:
Rather than instantiating myDate here, which means as soon as the code runs your internal date is set, this value should only be set when one of the module's methods are called.